rdflib/__init__.py

```diff
@@ -83,6 +83,34 @@
     "util",
 ]
 
+import sys
```

Comment:
I don't think this is appropriate. The library should just log, setting up handlers should be up to users of the library.